pybleau/app/model/dataframe_plot_manager.py

```diff
@@ -811,25 +816,22 @@ def _request_template_name_with_desc(self, desc: PlotDescriptor) -> \
             name or makes a new one, returns that name as a str.
         """
         options = list(self.custom_configs.keys())
+        template_name = desc.plot_title
+        select = TemplatePlotNameSelector(
+            new_name=template_name,
+            plot_types=options,
+            model=self.template_manager,
+        )
+
         basis = desc.plot_config.source_template
-        if basis is not None and basis in options:
-            select = TemplatePlotNameSelector(
-                new_name="",
-                plot_types=options,
-                selected_string=basis,
-                model=self.template_manager,
-                replace_old_template=True
-            )
-        else:
-            if basis is not None and basis not in options:
-                logger.warning(f'"{desc.plot_title}" is a template, but does '
+        if basis != "":
+            if basis not in options:
+                logger.warning(f'"{basis}" is a template, but does '
```

Comment:
Should we do something here? Use the basis to set the `new_name`?

Reply:
Yes, will do that.